airflow/dag_processing/collection.py

```diff
@@ -36,6 +36,7 @@
 
 from airflow.assets import Asset, AssetAlias
 from airflow.assets.manager import asset_manager
+from airflow.models import Trigger
```

Comment:
In core Airflow I prefer to always import from the actual definition location instead, i.e. `from airflow.models.trigger import Trigger`.

Reply:
Good point, feel like there should be a pre-commit for that